hw2/03-count.js

```diff
@@ -1,3 +1,62 @@
-// Add your code here
+// Using a const to just load it from here instead of fixed on the page.
+// Doing this so that it's easier to refresh the context of what we are searching
+// and not having to worry about story and moving it all over the place.
+const searchContext = `
+It was the best of times, it was the worst of times, it was
+the age of wisdom, it was the age of foolishness, it was the
+epoch of belief, it was the epoch of incredulity, it was the
+season of Light, it was the season of Darkness, it was the
+spring of hope, it was the winter of despair, we had
+everything before us, we had nothing before us, we were all
+going direct to Heaven, we were all going direct the other
+way—in short, the period was so far like the present period,
+that some of its noisiest authorities insisted on its being
+received, for good or for evil, in the superlative degree of
+comparison only.
+`;
```

Comment:
I can imagine your const variable here would be more efficient as well.  Just make sure the HTML page content never changes!